docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -37,17 +37,19 @@ Let's get started with a simple "Hello World" program on .NET Core:
     * Alternatively, you can select **File** > **Open Folder** from the main menu to open your project folder.
 
 2. Initialize a C# project:
-    * Open the Integrated Terminal from Visual Studio Code by typing <kbd>CTRL</kbd>+<kbd>\`</kbd> (backtick).
+    * Open the Integrated Terminal from Visual Studio Code by typing <kbd>CTRL</kbd>+<kbd>\`</kbd> (backtick). Alternatively, you can select **View** > **Integrated Terminal** from the main menu.
     * In the terminal window, type `dotnet new console`.
     * This creates a `Program.cs` file in your folder with a simple "Hello World" program already written, along with a C# project file named `HelloWorld.csproj`.
 
   ![The dotnet new command](media/with-visual-studio-code/dotnetnew.png)
 
 3. Resolve the build assets:
 
-    * Type `dotnet restore`. Running `dotnet restore` gives you access to the  required .NET Core packages that are needed to build your project.
+    * For **.NET Core 1.1**, type `dotnet restore`. Running `dotnet restore` gives you access to the  required .NET Core packages that are needed to build your project.
 
-  ![The dotnet restore command](media/with-visual-studio-code/dotnetrestore.png)
+   ![The dotnet restore command](media/with-visual-studio-code/dotnetrestore.png)
+
+    * For **.NET Core 2.0**, this step is optional. The `dotnet restore` command executes automatically when a new project is created.
```

Comment:
I'd add that `dotnet restore` runs as a dependent task on each build, if necessary.